tracer/src/Datadog.Trace/ClrProfiler/Instrumentation.cs

```diff
@@ -371,7 +373,7 @@ private static void InitRemoteConfigurationManagement(Tracer tracer)
                         var rcmApi = RemoteConfigurationApiFactory.Create(tracer.Settings.Exporter, rcmSettings, discoveryService);
                         var tags = GetTags(tracer, rcmSettings);
 
-                        var configurationManager = RemoteConfigurationManager.Create(discoveryService, rcmApi, rcmSettings, serviceName, tracer.Settings.Environment, tracer.Settings.ServiceVersion, tags);
+                        var configurationManager = RemoteConfigurationManager.Create(discoveryService, rcmApi, rcmSettings, serviceName, TraceUtil.NormalizeTag(tracer.Settings.Environment), tracer.Settings.ServiceVersion, tags);
```

Comment:
Should we normalize the `ServiceVersion` too? Prob unnecessary, but better to be safe?

Reply:
Good idea! Done in 07e24548631653b407159077cf3de015d03b5cf6